### ISSUE TITLE
[1.19.2] Recycling recipes are still added even though they're disabled in the config

### ISSUE THREAD

**Issue by vizthex123** [Qmunity/BluePower#623]
### What were you doing when the issue happened? ###
Just checking the recycling config to be sure it got properly disabled

### What did you expect to happen? ###
The recipes wouldn't get added

### What happened instead ###
The recipes got added

### Full Log and Crashlog on pastebin/gist ###
https://gist.github.com/vizthex123/efae0b4d21bf404826497752b2e57b4a

### Forge and Minecraft version ###
MC 1.19.2, Forge 43.3.2

### Bluepower version and build number ###
0.8.315-alpha-universal

**Comment by MoreThanHidden** [comment]
Usually this is caused by the json files that were generated still being in the world directory, I'll take a look though.

**Comment by vizthex123** [comment]
> Usually this is caused by the json files that were generated still being in the world directory, I'll take a look though.

Yeah, I figured that was the case at first - but making a new world also adds the recycling recipes, so I decided to report it (should've said so in the original report, sorry).

It'd be nice if it could delete existing recipes in a world if it was disabled too.

**Comment by MoreThanHidden** [comment]
Has been fixed in newer versions as I now use dynamic recipes, will see if I've backported that yet.
